cmd/juju/model/exportbundle.go

```diff
@@ -61,6 +67,7 @@ func (c *exportBundleCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.ModelCommandBase.SetFlags(f)
 	f.StringVar(&c.Filename, "filename", "", "Bundle file")
 	f.BoolVar(&c.includeCharmDefaults, "include-charm-defaults", false, "Whether to include charm config default values in the exported bundle")
+	f.BoolVar(&c.includeSeries, "export-with-series", false, "Comaptibility option. Set to include series in the bundle alongside bases")
```

Comment:
I think "--include-series" as the arg name is better than "--export-wth-series". The latter is clumsy and the former conveys the intent needed. We know it's being exported so no need to include that again in the arg name

Reply:
agree, plus it's consistent with `--include-charm-defaults`